win32/src/win32apimodule.cpp

```diff
@@ -4778,10 +4778,7 @@ static PyObject *PySetSystemTime(PyObject *self, PyObject *args)
     {
         return ReturnAPIError("SetSystemTime");
     }
-    else
-    {
-        return Py_BuildValue("i", result);
-    }
+    else { return Py_BuildValue("i", result); }
```

Comment:
this formatting seems off?

Reply:
Indeed.
Btw I just noticed that the clang-format CI check shows formatting issues, but doesn't actually fail the checkers job.

Anyway, this can be run locally with either:
```shell
pip install pre-commit
pre-commit run clang-format --all
# or
pip install clang-format
clang-format --Werror --dry-run $(git ls-files '*.cpp' ':!:com/win32comext/mapi/src/MAPIStubLibrary/')
```

Reply:
Yes, I was playing with the formatter, as I didn't like some of the rules, and I forgot about this part. Thanks for fixing!.

Reply:
does this still need to be fixed here?